mcs/class/Mono.Posix/Mono.Unix.Native/Syscall.cs

```diff
@@ -727,6 +727,194 @@ public enum MremapFlags : ulong {
 		MREMAP_MAYMOVE = 0x1,
 	}
 
+	[Map]
+	[CLSCompliant (false)]
+	public enum SockType : int {
```

Comment:
The Problem with this is that it will create a conflict for users who import both System.Net.Sockets and Mono.Unix.Native user SocketType without qualifying it. (Some files in the mono repository do this.) If you want to I can change it anyway (and fix the stuff in the mono repository).